scripts/verify.py

```diff
@@ -45,16 +45,13 @@ def main(checks):
     checks = ['pytest', 'pylint', 'mypy', 'build-docs', 'check-docs']
 
     parser = argparse.ArgumentParser()
-
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument('--all', action='store_true', help='Run all tests.')
-    group.add_argument('--checks', type=str, nargs='+', choices = checks)
+    parser.add_argument('--checks', type=str, required=False, nargs='+', choices = checks)
```

Comment:
2 nits:

(1) spaces around `choices = checks`
(2) isn't `required=False` the default for `--` arguments?